### PR TITLE
ztp: CNF-14990: Fix golang ocp builder image in resource generator

### PR DESCRIPTION
**Pull request by fontivan**
- Should match the golang and ocp versions used in current release

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,6 +5,7 @@ module github.com/openshift-kni/cnf-features-deploy
 // This should also be matched within this project at:
 //   - cnf-tests/Dockerfile*
 //   - openshift-ci/Dockerfile*
+//   - ztp/resource-generator/Containerfile
```
Comment (fontivan) [inline go.mod:8]: I think adding this comment means I need a /approve for `infra` instead of `ztp` which is unfortunate

/cc @imiller0 @ffromani

**Review by lack — APPROVED** [review]

**Review by ffromani — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**